packages/client/src/views/SysAdmin/Team/user/UserList.tsx

```diff
@@ -553,10 +553,12 @@ function UserListComponent(props: IProps) {
         (user: User | null, index: number) => {
           if (user !== null) {
             const name =
-              (user &&
-                user.name &&
-                ((createNamesMap(user.name)[intl.locale] as string) ||
-                  (createNamesMap(user.name)[LANG_EN] as string))) ||
+              (user.name &&
+                intl.formatMessage(constantsMessages.humanName, {
+                  firstName: user.name[0].firstNames,
+                  middleName: user.name[0].middleName,
+                  lastName: user.name[0].familyName
+                })) ||
```

Comment:
It would be better to have a utility function similar to the `createNamesMap`, something like this:
```
function internationalizedName(name: HumanName[], locale: Locale): { 
  firstName: string
  middleName: string
  lastName: string
}
```
It would try to find the name in the given locale, falling back to 'en' in case it does not find it

Reply:
I think this is where I was getting confused. It looks like the translations are passed through from `IntlProvider` and also pulled from `country-config`, now passing the locale implies that we have access to all different translations and dynamically changes languages. Do you know where I can get this?

Reply:
You can just pass in `intl.locale` to the function. As for how to resolve that, take a look at the current implementation of the `createNamesMap` function, it should give you an idea.

Reply:
Do we really support names in multiple languages 🤔 If not, then I'd aim to simplify as much as we can. IMO best would be a component like 

```
<HumanName name={user.name} />
```

component that could use the `useIntl` hook. Passing the intl object as a function parameter feels like a little upside down

Reply:
Hi @rikukissa on the system user we only support one language, English and on the Birth and Death forms we looks like we are only ask `First names` and `Surname`. Does this confirm that we don't support name in multiple languages for `Birth`, `Marriage` and `Death` as well?

Reply:
Yes, I think that's a safe assumption